lib/PGUtil.pm

```diff
@@ -92,7 +92,17 @@ sub pretty_print_html {    # provides html output -- NOT a method
     $level--;
     return "PGalias has too much info. Try \$PG->{PG_alias}->{resource_list}" if ref($r_input) eq 'PGalias';  # PGalias just has too much information
     return 'too deep' unless $level > 0;  # only print four levels of hashes (safety feature)
-    my $out = '';
+	my $out = '';
+	    # protect against modules defined in Safe which can't find their stringify procedure.
+		$r_input = $r_input//'';
+		eval { "$r_input" };
```

Comment:
Line 98 here is giving me the compilation error: Useless use of string in void context at /opt/webwork/pg/lib/PGUtil.pm line 98.